draft-ietf-moq-transport.md

```diff
@@ -2618,19 +2654,29 @@ effect on outstanding subscriptions.
 ### Subgroup Header
 
 When a stream begins with `SUBGROUP_HEADER`, all Objects on the stream
-belong to the track requested in the Subscribe message identified by `Track Alias`
-and the subgroup indicated by 'Group ID' and `Subgroup ID`.
+belong to the track requested in the subscription identified by `Subscription
+Identifier` and the subgroup indicated by 'Group ID' and `Subgroup ID`.
 
 ~~~
 SUBGROUP_HEADER {
-  Track Alias (i),
+  Subscription Identifier (i),
   Group ID (i),
   Subgroup ID (i),
   Publisher Priority (8),
 }
 ~~~
 {: #object-header-format title="MOQT SUBGROUP_HEADER"}
 
+* Subscription Identifier: the Subscriber or Publisher chosen Subscription
+  Identifier indicating the subscription this Subgroup belongs to. If an
+  endpoint receives a subgroup with an unknown Subscriber-Chosen Subscription
+  Identifier, it MUST close the connection with a Protocol Violation.  If it
+  receives a subgroup with an unknown Publisher-Chosen Subscription Identifier,
+  it MAY abandon the stream, or choose to buffer it for a brief period to handle
+  reordering with the control message that establishes the PSID.  The endpoint
+  SHOULD NOT release stream flow control beyond the SUBGROUP_HEADER until the
+  PSID has been established.  TODO: talk about possible deadlocks.
```

Comment:
```suggestion
  endpoint receives a subgroup with aSubscriber-Chosen Subscription
  Identifier that does not correspond to a subscription it initiated, it MUST
  close the connection with a Protocol Violation.  If it receives a subgroup
  with an unknown Publisher-Chosen Subscription Identifier, it MAY
  abandon the stream, or choose to buffer it for a brief period to handle
  reordering with the control message that establishes the PSID.  The endpoint
  SHOULD NOT release stream flow control beyond the SUBGROUP_HEADER until the
  PSID has been established.  TODO: talk about possible deadlocks.
```